syntaxes/src/expression/grammar.ts

```diff
@@ -0,0 +1,886 @@
+import { GrammarDefinition } from '../types';
+
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export const expression: GrammarDefinition = {
+  scopeName: 'expression.ng',
+  injectionSelector: 'L:text.html -comment',
+  patterns: [{ include: '#ngExpression' }],
+  repository: {
+    ngExpression: {
+      name: 'meta.expression.ng',
+      patterns: [
+        {
+          include: '#tsString',
```

Comment:
```suggestion
          include: '#string',
```

Reply:
changed all of the references from tsString to string

Reply:
I'm skipping commiting the suggestion here as there are the references to the pattern in the file